README.md

```diff
@@ -1,26 +1,48 @@
 <!--
-Avoid using this README file for information that is maintained or published elsewhere, e.g.:
-
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
-
-Use links instead.
+[![CharmHub Badge](https://charmhub.io/penpot/badge.svg)](https://charmhub.io/penpot)
+[![Publish to edge](https://github.com/canonical/penpot-operator/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/penpot-operator/actions/workflows/publish_charm.yaml)
+[![Promote charm](https://github.com/canonical/penpot-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/penpot-operator/actions/workflows/promote_charm.yaml)
+[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
 -->
 
-# is-charms-template
-
-Charmhub package name: operator-template
-More information: https://charmhub.io/is-charms-template
+A Juju charm deploying and managing [Penpot](https://penpot.app) on Kubernetes. Penpot is the
```

Comment:
Nitpick: The first sentence reads a bit strange to me. I would personally rephrase to say either "A Juju charm that deploys and manages Penpot..." or "A Juju charm to deploy and manage Penpot...".

In any event, it doesn't bother me too much.

Reply:
I like it too, have changed this.